docs/source/guides/contribution_guide.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+![NuCypher Unicorn](https://cdn-images-1.medium.com/max/800/1*J31AEMsTP6o_E5QOohn0Hw.png)
```

Comment:
Yes, awesome :) If we want more of these, @Carusosa can get a bunch more!

Reply:
Oh - Yes. :+1: